### PR TITLE
PDFill - Add 15.0 Build 3 and remove old version

### PR DESCRIPTION
**Pull request by Witchilich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
There is only one package url, so you have to remove old manifest with each new version.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18153)

### CODE REVIEW

**Comment by Witchilich** [comment]
No reveiwers?

File: manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: PlotSoft.PDFill
+PackageVersion: 15.0 Build 3
```
Comment (OfficialEsco) [inline manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.installer.yaml:2]:
```suggestion
PackageVersion: 15.0
```

File: manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: PlotSoft.PDFill
+PackageVersion: 15.0 Build 3
```
Comment (OfficialEsco) [inline manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.locale.en-US.yaml:2]:
```suggestion
PackageVersion: 15.0
```

File: manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: PlotSoft.PDFill
+PackageVersion: 15.0 Build 3
```
Comment (OfficialEsco) [inline manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.yaml:2]:
```suggestion
PackageVersion: 15.0
```

File: manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+PackageLocale: en-US
+Publisher: PlotSoft LLC
+PublisherUrl: http://www.plotsoft.com/
+PublisherSupportUrl: http://www.plotsoft.com/support.html
+PackageName: PDFill
```
Comment (OfficialEsco) [inline manifests/p/PlotSoft/PDFill/15.0 Build 3/PlotSoft.PDFill.locale.en-US.yaml:7]:
```suggestion
PackageName: PDFill FREE PDF Editor Basic
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image
Remember to change the folder name

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
As i said rename the folder :)
` Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\p\PlotSoft\PDFill\15.0 Build 3', expected 'manifests\p\PlotSoft\PDFill\15.0'`

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**